tox.ini

```diff
@@ -15,17 +15,17 @@ commands = yapf -dr . {posargs}
 [testenv:isort]
 allowlist_externals = isort
 skip_install = true
-commands = isort -c ./kannon ./test ./example {posargs}
+commands = isort -c ./kannon ./test ./integration_test ./example {posargs}
```

Comment:
I think it can be simpler like `isort -c ./kannon . {posargs}` , but need to take care to ignore `./.venv`. 

I skipped to changing style in this PR and just added new dir.